File: app/src/main/java/com/wealoha/social/beans/BlackListResult.java
package com.wealoha.social.beans;

import java.util.List;
import java.util.Map;

import com.wealoha.social.beans.imagemap.HasImageMap;

public class BlackListResult extends ResultData implements HasImageMap {

	public List<User> list;
	public String nextCursorId;
	private Map<String, Image> imageMap;

	@Override
	public Map<String, Image> getImageMap() {
		return imageMap;
	}
}

File: app/src/main/java/com/wealoha/social/beans/BlackListResult.kt
package com.wealoha.social.beans

import com.wealoha.social.beans.imagemap.HasImageMap

data class BlackListResult(
    var list: List<User>? = null,
    var nextCursorId: String? = null,
    override val imageMap: Map<String, Image>? = null

) : ResultData(), HasImageMap {

}